lib/Conversion/TTIRToTTIRDecomposition/TTIRToTTIRDecomposition.cpp

```diff
@@ -397,7 +397,107 @@ struct ConvolutionToConv2dPattern
                                              adaptor.getOperandConstraints());
 
     rewriter.replaceOp(op, output);
+    return success();
+  }
+};
+//===----------------------------------------------------------------------===//
+// Gather passes
```

Comment:
We are not defining a pass, we are defining a matching pattern, hence can we please rename this to something different? For example:
Gather patterns